test/e2e/tests/send-eth.spec.js

```diff
@@ -43,6 +43,7 @@ describe('Send ETH from inside MetaMask using default gas', function () {
         await inputAmount.press(driver.Key.BACK_SPACE);
         await inputAmount.press(driver.Key.BACK_SPACE);
         await inputAmount.press(driver.Key.BACK_SPACE);
+        await driver.delay(regularDelayMs);
```

Comment:
Curious that this delay was needed :thinking: Does the error message disappear more slowly now?

Reply:
It doesn't *seem* to be slower but it must be imperceptibly slower. It would succeed sometimes without the delay.